python/dgl/transforms/module.py

```diff
@@ -544,7 +544,7 @@ class AddSelfLoop(BaseTransform):
      >>> g.edata['he'] = torch.arange(3).float().reshape(-1, 1)
      >>> new_g = transform(g)
      >>> print(new_g.edges())
-     (tensor([1, 0, 1, 2]), tensor([2, 0, 1, 2]))
+     (tensor([0, 0, 2, 0, 1, 2]), tensor([2, 1, 0, 0, 1, 2]))
```

Comment:
Can you also fix the issues on the other doc string for this method?

e.g. L560 to L567 is misaligned, such that the render [under case 2](https://docs.dgl.ai/generated/dgl.transforms.AddSelfLoop.html) is broken.

Reply:
OK, I will try to fix them.